build.sbt

```diff
@@ -5,6 +5,7 @@ val akkaHttpVersion = "10.1.0"
 val akkaVersion = "2.5.11"
 val slickVersion = "3.2.1"
 val circeVersion = "0.11.0"
+val catsVersion = "1.6.0-RC1"
```

Comment:
why we want release candidate here? does it have any major features/fixes we need?

Reply:
it simply was the latest available at the time, as suggested on the website.
Good for you to notice, as they've bumped to 1.6.0 stable, so I can update